src/libcore/sync/atomic.rs

```diff
@@ -241,7 +257,7 @@ impl AtomicBool {
 
     /// Loads a value from the bool.
     ///
-    /// `load` takes an [`Ordering`] argument which describes the memory ordering
+    /// `load()` takes an [`Ordering`] argument which describes the memory ordering
```

Comment:
we don't include the `()` in function names; some of our docs do, but we decided against that style.